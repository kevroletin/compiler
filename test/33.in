Program xyz;
v{aaaaaaaaaaaaaaa}ar{ible} i: Integer;
	x,y,z: Real//yes, it's real!
;
begin
{


}
end.