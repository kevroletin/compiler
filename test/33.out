1:1 RESERVED_WORD Program
1:9 IDENTIFIER xyz
1:12 DELIMITER ;
2:1 RESERVED_WORD var
2:28 IDENTIFIER i
2:29 DELIMITER :
2:31 IDENTIFIER Integer
2:38 DELIMITER ;
3:2 IDENTIFIER x
3:3 DELIMITER ,
3:4 IDENTIFIER y
3:5 DELIMITER ,
3:6 IDENTIFIER z
3:7 DELIMITER :
3:9 IDENTIFIER Real
4:1 DELIMITER ;
5:1 RESERVED_WORD begin
10:1 RESERVED_WORD end
10:4 OPERATION .
10:5 END_OF_FILE 